Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -252,23 +252,28 @@ class AccountStatusSetting: WithAccountSetting {
 
     override var status: NSAttributedString? {
         if let account = profile.getAccount() {
+            let orange = UIColor(red: 255.0 / 255, green: 149.0 / 255, blue: 0.0 / 255, alpha: 1)
+            
             switch account.actionNeeded {
             case .none:
                 return nil
             case .needsVerification:
-                return NSAttributedString(string: NSLocalizedString("Verify your email address.", comment: "Text message in the settings table view"), attributes: [NSForegroundColorAttributeName: UIConstants.TableViewRowTextColor])
+                let string = NSLocalizedString("Verify your email address.", comment: "Text message in the settings table view")
+                let range = NSRange(location: 0, length: string.characters.count)
+                let attrs = [NSForegroundColorAttributeName: orange]
+                let res = NSMutableAttributedString(string: string)
+                res.setAttributes(attrs, range: range)
+                return res
             case .needsPassword:
                 let string = NSLocalizedString("Enter your password to connect.", comment: "Text message in the settings table view")
                 let range = NSRange(location: 0, length: string.characters.count)
-                let orange = UIColor(red: 255.0 / 255, green: 149.0 / 255, blue: 0.0 / 255, alpha: 1)
                 let attrs = [NSForegroundColorAttributeName: orange]
                 let res = NSMutableAttributedString(string: string)
                 res.setAttributes(attrs, range: range)
                 return res
             case .needsUpgrade:
                 let string = NSLocalizedString("Upgrade Firefox to connect.", comment: "Text message in the settings table view")
                 let range = NSRange(location: 0, length: string.characters.count)
-                let orange = UIColor(red: 255.0 / 255, green: 149.0 / 255, blue: 0.0 / 255, alpha: 1)
```

Comment:
You've started factoring `orange` out of these cases.

Can you factor out anything else?

e.g. 
```swift
let color: UIColor
let string: NSString?

switch account.actionNeeded {
    // set the string and color
}

if let string = string {
    // do the range, attrs and res
    return res
}
return nil
```